rocky-linux/createhdds
# Split kickstart files for Rocky 8 and 9

## Description

This PR makes sure that the individual release version builds use the correct repo path, by splitting the kickstart files as possible in the createhdds.py tool!

Notice that most 9 files are still called without versioning, to make sure different versions still have something to grab.

There is also another change needed in the os-autoinst repo to make the kickstart file support bundle for the support server working again, will arrive there soon in the open PR.

## How to test

Run the creation of all images `./createhdds.py -t all`
Verify afterwards that the images work as they should!

## Checklist:
